lib/open_project/static/links.rb

```diff
@@ -177,6 +177,10 @@ def static_links
               href: "https://www.openproject.org/docs/release-notes/",
               label: :label_release_notes
             },
+            release_notes_14_0_1: {
+              href: "https://www.openproject.org/docs/release-notes/14-0-1/",
+              label: "Release notes for OpenProject 14.0.1"
+            },
```

Comment:
Yes, it's not used. I saw some other links having a non-translated non-used label for links. I added it so that the "interface" of the links is respected. Should I remove it?

Reply:
I don't see value in having it other than maybe making it clearer what the string actually is and for that, I would use a comment. So yes, I would remove it.